src/Game.cpp

```diff
@@ -849,7 +850,7 @@ bool Game::CanLoadGame(const std::string &filename)
 {
 	Output("Game::CanLoadGame('%s')\n", filename.c_str());
 	auto file = FileSystem::userFiles.ReadFile(FileSystem::JoinPathBelow(Pi::SAVE_DIR_NAME, filename));
-	if (!file) 
```

Comment:
Yes, I know... So: if this is critical I will "convince" my editor to leave things as they are, but for now *he* is working more hard then me ;-)

Let me know...